Anlab.Mvc/Views/Order/Index.cshtml

```diff
@@ -22,12 +22,12 @@
             <td>
                 <select name="status" id="status">
                     <option value="All">Show All</option>
-                    <option value="Created">Created</option>
-                    <option value="Confirmed">Confirmed</option>
-                    <option value="Received">Received</option>
-                    <option value="Finalized">Finalized</option>
-                    <option value="Accepted">Accepted</option>
-                    <option value="Complete">Complete</option>
+                    <option value="@OrderStatusCodes.Created">@OrderStatusCodes.Created</option>
+                    <option value="@OrderStatusCodes.Confirmed">@OrderStatusCodes.Confirmed</option>
+                    <option value="@OrderStatusCodes.Received">@OrderStatusCodes.Received</option>
+                    <option value="@OrderStatusCodes.Finalized">@OrderStatusCodes.Finalized</option>
+                    <option value="@OrderStatusCodes.Accepted">@OrderStatusCodes.Accepted</option>
+                    <option value="@OrderStatusCodes.Complete">@OrderStatusCodes.Complete</option>

```

Comment:
@laholstege I think here either create a select list in order status codes or create a new array property (call it All?) that is just an array of all valid status codes.  Could then easily extend to have other arrays later.  That way on both of these pages you can keep your select list but then foreach your <option> tags so that we don't have to know where to change things when new statuses are added